physical_infra_provider_configuration/overview/topics/architecture_overview.adoc

```diff
@@ -1 +1,3 @@
-=== Architecture Overview
\ No newline at end of file
+=== Architecture Overview
+
+The Lenovo XClarity Provider communicates with instances of the Lenovo XClarity appliance to retrieve information about managed resources, monitor events and carry out actions.  Resource details are aggregated in the ManageIQ database and presented to the user through the ClouForms Compute -> Physical Hardware --v.    The CloudForms user interface generates the relationship between existing harware details and the virtual host details that form the customer's solution.  
```

Comment:
Probably didn't mean to include "--v" in "ClouForms Compute -> Physical Hardware --v" and "d" is missing in "ClouForms". Also, there is extra whitespace at the end of the sentence.